cmd/demo/node.go

```diff
@@ -443,14 +461,23 @@ func (n *node) Info(args []string) error {
 	n.log.Traceln("Info...")
 
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', tabwriter.Debug)
-	fmt.Fprintf(w, "Peer\tPhase\tVersion\tMy Ξ\tPeer Ξ\t\n")
+	fmt.Fprintf(w, "Peer\tPhase\tVersion\tMy Ξ\tPeer Ξ\tMy On-Chain Ξ\tPeer On-Chain Ξ\t\n")
 	for alias, peer := range n.peers {
+		ctx, cancel := context.WithTimeout(context.Background(), config.Chain.DeployTimeout)
+		defer cancel()
+		onChainBalListWei, err := getOnChainBal(ctx, n.onChain.Address(), peer.perunID)
+		if err != nil {
+			return err
+		}
+
 		if peer.ch == nil {
-			fmt.Fprintf(w, "%s\t%s\t\n", alias, "Connected")
+			fmt.Fprintf(w, "%s\t%s\t \t \t \t%s\t%s\t\n", alias, "Connected",
+				weiToEther(onChainBalListWei[0]).String(), weiToEther(onChainBalListWei[1]).String())
 		} else {
-			my, other := peer.ch.GetBalances()
-			fmt.Fprintf(w, "%s\t%v\t%d\t%v\t%v\t\n",
-				alias, peer.ch.Phase(), peer.ch.State().Version, my, other)
+			myBalWei, peerBalWei := peer.ch.GetBalances()
+			fmt.Fprintf(w, "%s\t%v\t%d\t%s\t%s\t%s\t%s\t\n",
+				alias, peer.ch.Phase(), peer.ch.State().Version, weiToEther(myBalWei).String(), weiToEther(peerBalWei).String(),
+				weiToEther(onChainBalListWei[0]).String(), weiToEther(onChainBalListWei[1]).String())
```

Comment:
Use `%v` for stringers.

Reply:
use variadic `weiToEth`.

Reply:
*big.Float doesn't implemet Stringer interface, String() should be called explicitly. See [here](https://golang.org/pkg/math/big/#Float.String).

Reply:
This only states that you cant use `%s`, excerpt from [go-doc](https://golang.org/pkg/math/big/) (search for *stringer*):  
```pre
… *Float values implement the Stringer interface for a (default) string representation …
```  
so `%v` works.

Reply:
The comment there refers to `Float.Format`. `big.Float` still implements the `fmt.Stinger` interface, so I think it works to call `fmt.Fprintf("%v", f)` on a `f *big.Float`. I also tried it out here: https://play.golang.org/p/h_KUgOq_iY2
So please try it out, I am curious. We also often print `big.Int`s with `%v`.

Reply:
Addressed in this commit - 806b792.